README.md

```diff
@@ -66,39 +66,46 @@ Through the Storefront, you can change the minicart's behavior and interface. Ho
-| `label`            | [index](https://github.com/vtex-apps/minicart/blob/master/react/index.js)            | Minicart icon label                                    |
-| `badge`            | [index](https://github.com/vtex-apps/minicart/blob/master/react/index.js)            | Minicart badge with the product quantity on it         |
-| `arrowUp`          | [Popup](https://github.com/vtex-apps/minicart/blob/master/react/components/Popup.js)            | Popup box arrow                                        |
-| `box`              | [Popup](https://github.com/vtex-apps/minicart/blob/master/react/components/Popup.js)            | The main container of the popup                        | 
-| `sidebarHeader`    | [Sidebar](https://github.com/vtex-apps/minicart/blob/master/react/components/Sidebar.js)          | Minicart sidebar header container                      |
-| `sidebar`          | [Sidebar](https://github.com/vtex-apps/minicart/blob/master/react/components/Sidebar.js)          | Minicart sidebar main container                        |
-| `sidebarOpen`      | [Sidebar](https://github.com/vtex-apps/minicart/blob/master/react/components/Sidebar.js)          | Active when the sidebar is opened                      |
-| `content`          | [MinicartContent](https://github.com/vtex-apps/minicart/blob/master/react/components/MiniCartContent.js)  | The container for the Minicart contents                            |
-| `contentSmall`     | [MinicartContent](https://github.com/vtex-apps/minicart/blob/master/react/components/MiniCartContent.js)   | The container for the Minicart contents when on desktop size                     |
-| `contentLarge`     | [MinicartContent](https://github.com/vtex-apps/minicart/blob/master/react/components/MiniCartContent.js)  | The container for the Minicart contents when on mobile size                     |
-| `contentDiscount`  | [MinicartFooter](https://github.com/vtex-apps/minicart/blob/master/react/components/MiniCartFooter.js)   | The total discount on the minicart footer              | 
-| `contentPrice`     | [MinicartFooter](https://github.com/vtex-apps/minicart/blob/master/react/components/MiniCartFooter.js)   | Total price of the products on the minicart footer     |
-| `contentFooter`    | [MinicartFooter](https://github.com/vtex-apps/minicart/blob/master/react/components/MiniCartFooter.js)   | The minicart footer main container                     |
+### Usage
 
-To override the default CSS, you need to import `styles` on your manifest:
+To use this CSS API, you must add the `styles` builder and create an app styling CSS file.
+
+1. Add the `styles` builder to your `manifest.json`:
 
 ```json
   "builders": {
     "styles": "1.x"
   }
 ```
 
-Also, create a `vtex.minicart.css` file in `styles/css` for your handlers customization.
+2. Create a file called `vtex.minicart.css` inside the `styles/css` folder. Add your custom styles:
+
+```css
+.container {
+  color: red;
```

Comment:
Add an example that change padding or margin, this example encourage the partners to not use tachyons generator.